outline.md

```diff
@@ -51,10 +51,10 @@ Modules allow you to divide logical portions of code into smaller, functional pi
 Now that you have an Ember-CLI generated project, let's step inside and start exploring:
 
     $ cd workshop
-    $ ls
-    Brocfile.js             bower.json              node_modules            testem.json
-    README.md               bower_components        package.json            tests
-    app                     config                  public                  vendor
+    $ ls -a
+    bower_components  Brocfile.js    .ember-cli  .jshintrc     public       tests
+    bower.json        config         .git        node_modules  README.md    .travis.yml
+    app  .bowerrc          .editorconfig  .gitignore  package.json  testem.json  vendor
 
```

Comment:
It's because I used `ls -a` instead of `ls -A` and then I removed the `.` and `..` entries.  I just pushed a fix.

We can replace with `ls -1a` if you prefer.